src/extension/prompts/node/agent/fileLinkificationInstructions.tsx

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { PromptElement } from '@vscode/prompt-tsx';
+import { Tag } from '../base/tag';
+
+export class FileLinkificationInstructions extends PromptElement<{}> {
+	render() {
+		return <Tag name='file_linkification'>
```

Comment:
Just a gut feeling but this seems pretty verbose and I'm not sure how well models will follow the good/bad method written this way